docs/docs/installation/configuration/vcluster.md

```diff
@@ -5,14 +5,9 @@ comments: true
 # vCluster installation
 
 Keptn running on Kubernetes versions 1.26 and older
-uses a custom
-[scheduler](../../components/scheduling.md),
-so it does not work with
+does not work with
```

Comment:
I think we should change this since Keptn is not compatible with k8s >1.27.
I don't think we need to talk about earlier k8s versions